x-pack/plugins/lens/public/visualizations/datatable/components/table_basic.test.tsx

```diff
@@ -545,53 +545,6 @@ describe('DatatableComponent', () => {
     });
   });
 
-  test('it detect last_value filtered metric type', () => {
```

Comment:
🎉 This is test is not needed anymore as it was added to guard against the buggy response we were getting. Now, as we get a proper response (number type is always number) we don't have to test this.